src/Raven.Server/RavenServer.cs

```diff
@@ -2490,8 +2509,9 @@ private bool TryAuthorize(RavenConfiguration configuration, Stream stream, TcpCo
                             throw new InvalidOperationException("Unknown operation " + header.Operation);
                     }
                 case AuthenticationStatus.UnfamiliarIssuer:
-                    msg = $"The client certificate {certificate.FriendlyName} is not registered in the cluster. Tried to allow the connection implicitly based on the client certificate's Public Key Pinning Hash but the client certificate was signed by an unknown issuer - closing the connection. " +
-                          $"To fix this, the admin can register the pinning hash of the *issuer* certificate: '{auth.IssuerHash}' in the '{RavenConfiguration.GetKey(x => x.Security.WellKnownIssuerHashes)}' configuration entry. Alternatively, the admin can register the actual certificate ({certificate.FriendlyName} '{certificate.Thumbprint}') explicitly in the cluster.";
+                    msg = $"The client certificate {certificate.FriendlyName} is not registered in the cluster. " +
+                          "Tried to allow the connection implicitly based on the client certificate's Public Key Pinning Hash but the client certificate was signed by an unknown issuer - closing the connection. " +
```

Comment:
Do we need to update this error? IIRC we won't use public key pinning anymore.

Reply:
No, there is still public key pinning hash, we just don't support registering by public key for well known